src/resources/views/configuration/settings/view.blade.php

```diff
@@ -234,4 +234,24 @@ class="form-control input-md" value="{{ setting('analytics_id', true) }}" disabl
     </div>
   </div>
 
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">{{ trans('web::seat.tp_versions') }}</h3>
+    </div>
+    <div class="panel-body">
+
+      <dl>
+
+        <dt>Eve Online SDE</dt>
+        <dd>
+          <ul>
+            <li>{{ trans('web::seat.installed') }}: <b>{{ Seat\Services\Settings\Seat::get('installed_sde') }}</b></li>
```

Comment:
Use the [setting helper](https://github.com/eveseat/services/blob/master/src/Helpers/helpers.php#L187) please.